modxcms/revolution
# Fix missed Developer Access Policy

### What does it do?
It adds to the map array in the special resolver missed `Developer` access policy.

### Why is it needed?
Such policy actually exists, but in case it does not have a linked access policy template, it is hidden in the list of policies. For access policy template is a mandatory field and it does not possible to create a Policy without Template. But when you try to create a policy with the name Developer, it will raise the issue that such a policy already exists. It may confuse people.

### How to test
- checkout code
- build the core package by running `php _build/transport.core.php`
- run setup and upgrade the installation
- check and ensure that Developer policy exists in the list with policies and Administrator template attached

### Related issue(s)/PR(s)
#15431
